### PR TITLE
feat: Add support for multiple profile reports

### LINKED ISSUE #45: Unable to see graphs when multiple profile results are available
If you run an ECL job where there are multiple data profiling results, no result graphs show up in the Resources tab (though there may be a lot of hidden content, based on how the scroll bar looks).

### PR DESCRIPTION
**Pull request by jbrundage**
Fixes #45

Signed-off-by: Jaman Brundage <jbrundage372@gmail.com>

### CODE REVIEW

**Comment by jbrundage** [comment]
@GordonSmith @dcamper with this PR multiple profile reports will display as tabs using the `result.Name` as the tab label

image

**Comment by dcamper** [comment]
Excellent!

@GordonSmith Please review when you get a chance.

I will probably combine this PR with the previous into a 1.5.1 release.

File: report/src/index.ts [hunk 1]
```diff
@@ -28,28 +29,85 @@ const config = {
...
+                    }
+                });
+            }
+        });
+        return this;
+    }
+}
+export interface ReportTabs{
+    hideSingleTabs_default(): boolean;
+    hideSingleTabs_default(_: boolean): this;
```
Comment (GordonSmith) [inline report/src/index.ts]: `_default` variants are not needed.

File: report/src/index.ts [hunk 2]
```diff
@@ -28,28 +29,85 @@ const config = {
...
+        const urlParts = this._espUrl.split("/WsWorkunits/res/");
+        const baseUrl = urlParts[0];
+        const urlParts2 = urlParts[1].split("/report/res/index.html");
+        const wuid = urlParts2[0];
+        this._wu = Workunit.attach({ baseUrl }, wuid);
+    }
+    render(callback?: (w: Widget) => void): this {
+        this._wu.fetchResults().then(results => {
+            if(this._results.length !== results.length) {
+                this._results = results;
```
[the remark this answers is not in the document]
Comment (jbrundage) [reply report/src/index.ts]: I don't think so, the DockPanel handles the widget cleaning.

File: report/src/index.ts [hunk 3]
```diff
@@ -28,28 +29,85 @@ const config = {
...
+                                        if (callback) {
+                                            callback(this as any);
+                                        }
+                                    });
+                                }
+                            })
+                            ;
+                    }
+                })
+            } else {
```
[the remark this answers is not in the document]
Comment (jbrundage) [reply report/src/index.ts]: But `result.fetchRows()` is async, so that `super.render` would happen before the widgets are added.

**Comment by jbrundage** [comment]
@GordonSmith I've amended with several of your change requests. I removed the `count` concept (render is now called after each Report widget is added)

File: report/src/index.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (GordonSmith) [reply report/src/index.ts]: Yes I see that now...

**Pull request merged**